### ISSUE TITLE
Support WoW64 for Wine

### ISSUE THREAD

**Issue by Pobega** [Pobega/com.fightcade.Fightcade#11]
https://wiki.winehq.org/Building_Wine#Shared_WoW64

**Comment by Pobega** [comment]
Seems at one point Winepak got it working

https://github.com/winepak/winepak-sdk/blob/2ad32c34d8fc376ddaa0f7811a76a0e213921c02/org.winepak.Sdk-staging.json#L128

**Comment by Pobega** [comment]
I have a WIP commit that I think would work, but I'm running into an issue where wine32 cannot see wine64's builddir as it gets cleaned when wine64 is done.

I can't find anything here that seems like it'd fix that. I tried making wine64 a module within wine32 and that didn't fix it either.

https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html

**Comment by Pobega** [comment]
Closing as won't fix, I'm going to use multiarch with wine32 instead. WoW64 proved to be too hard to build properly in the Flatpak build environment, though [this PR](https://github.com/Pobega/com.fightcade.Fightcade/pull/13) shows the things I tried to get it to build.

The issue is that when building a module you cannot see the build directory for previous modules, so I was unable to get the `--with-wine64=` buildflag for WoW64 to work.

Closing in favor of https://github.com/Pobega/com.fightcade.Fightcade/pull/14
